src/site/content/en/blog/optimize-fid/index.md

```diff
@@ -0,0 +1,300 @@
+improvements in TBT usually correspond to improvements in FID. 
+
+The main cause of a poor FID is **heavy JavaScript execution**. Optimizing how JavaScript parses,
+compiles, and executes on your web page will directly reduce input delay.
+
+## Heavy JavaScript execution
+
+The browser cannot respond to user input while it's executing JavaScript. In other words, the
+browser can't respond to user interactions while the main thread is busy. To improve this:
+
++   [Reduce JavaScript execution time](#reduce-javascript-execution)
++   [Break up Long Tasks](#break-long-tasks)
++   [Optimize your page for interaction readiness](#optimize-interaction-readiness)
++   [Use a web worker](#web-worker)
+
+### Reduce JavaScript execution time {: #reduce-javascript-execution }
+
+Reducing the amount of JavaScript on your page reduces the amount of time that the browser needs to
+spend parsing, compiling, and executing JavaScript code. This speeds up how fast the browser can
+begin to respond to any user interactions.
+
+To reduce the amount of JavaScript executed on your page:
+
++   Minify and compress JavaScript files
++   Defer unused JavaScript
++   Minimize unused polyfills
+
+In addition to minimizing JavaScript, breaking up long-running code into smaller tasks can allow the
+browser to run input handlers sooner which can improve FID.
+
+### Minify and compress JavaScript files
+
+JavaScript can contain characters that are unnecessary for browsers but makes things easier to read
+during development. This includes spacing, indentation, comments, and long variable names.
+
+[Terser](https://github.com/terser/terser) supports ES6+ syntax, and can be used to minify modern
+JavaScript files without the need to transpile them. If you use a module bundler and want to include
+Terser in your toolchain:
+
++   webpack and Parcel already minify using Terser by default in `production` mode
++   If you use Rollup, include
+    [rollup-plugin-terser](https://www.npmjs.com/package/rollup-plugin-terser) as an output plugin
+
+In addition to minification, compress your JavaScript resources to minimize their delivery size. If
+possible, use
+[Brotli](https://opensource.googleblog.com/2015/09/introducing-brotli-new-compression.html) which
+provides better compression results than
+[Gzip](https://www.youtube.com/watch?v=whGwm0Lky2s&feature=youtu.be&t=14m11s) and [can be used in
+almost all newer browsers](https://caniuse.com/#feat=brotli).
+
+{% Aside %}
+For more details, refer to the [Minify and compress network payloads](/reduce-network-payloads-using-text-compression/) guide.
+{% endAside %}
+
+### Defer unused JavaScript
+
+By default all JavaScript is render-blocking. When the browser encounters a script tag that links to
+an external JavaScript file, it must pause what it's doing and download, parse, compile, and execute
+that JavaScript. Therefore you should only load the code that's needed for loading the page or
```

Comment:
The phrase "only **load** the code that's needed for **loading** the page" is a bit awkward. (Due to repeated "load").

How about: "...only load the code that's needed for the page or..."

Reply:
```suggestion
that JavaScript. Therefore you should only load the code that's needed for the page or
```